manticore/core/memory.py

```diff
@@ -312,22 +312,26 @@ def __setitem__(self, index, value):
             self._overlay[index] = value
 
     def __getitem__(self, index):
-        def get_byte_at_offset(offset):
```

Comment:
i think we probably should still keep this as a function, rather than copy/pasting the code. rather than being an inline function , it can be a private method.